compiler/luci-micro/standalone/CMakeLists.txt

```diff
@@ -7,14 +7,30 @@ set(BUILD_WHITELIST "dummy")
 add_subdirectory(${NNAS_ROOT}/infra/nncc ${CMAKE_CURRENT_BINARY_DIR}/nncc)
 
 set(ONE_COMPILER_SRC_DIR "${NNAS_PROJECT_SOURCE_DIR}/compiler")
+include_directories(${ONE_COMPILER_SRC_DIR}/luci-micro/mbed-os)
```

Comment:
this file and `set(BUILD_WHITELIST "dummy")` seems a hack. I think it would be better not to direct include compiler modules.

One way I could think of is to provide a script in `infra/nncc` with `BUILD_WHITELIST` for `micro`